### PR TITLE
Adding svt-av1-psy as a separate package

### PR DESCRIPTION
**Pull request by BlueSwordM**
Continuation of this PR: https://github.com/CachyOS/CachyOS-PKGBUILDS/pull/141

This time though, it's actually a separate PKGBUILD so that users not only have to manually install it, but it'll get updated separately I believe.

Any improvements welcome.

### CODE REVIEW

File: svt-av1-psy/.SRCINFO [hunk 1]
```diff
@@ -0,0 +1,17 @@
+pkgbase = svt-av1
```
Comment (ptr1337) [inline svt-av1-psy/.SRCINFO:1]: SRCINFO needs to be updated

File: svt-av1-psy/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  git describe --long --tags | sed 's/\([^-]*-g\)/r\1/;s/-/./g'
+}
+
+build() {
+  export LDFLAGS="$LDFLAGS -Wl,-z,noexecstack"
+  cmake -S svt-av1-psy -B build -G Ninja \
+    -DCMAKE_INSTALL_PREFIX=/usr \
+    -DBUILD_SHARED_LIBS=ON \
+    -DENABLE_AVX512=ON \
+    -DNATIVE=OFF
```
Comment (ptr1337) [inline svt-av1-psy/PKGBUILD:44]: I think -DNative can be completl removed, since its default off

**Review by ptr1337 — CHANGES_REQUESTED** [review]
LGTM, but check the two comments

File: svt-av1-psy/PKGBUILD (hunk 2 above)
Comment (BlueSwordM) [reply svt-av1-psy/PKGBUILD]: Removed -DNATIVE.

File: svt-av1-psy/.SRCINFO (hunk 1 above)
Comment (BlueSwordM) [reply svt-av1-psy/.SRCINFO]: Updated the SRCINFO.

**Pull request merged**